kong/runloop/handler.lua

```diff
@@ -204,6 +211,14 @@ local function csv(s)
 end
 
 
+local function try_incr_counter(key)
+  -- key must be DECLARATIVE_ROUTERS_REBUILD_COUNT_KEY or DECLARATIVE_PLUGINS_REBUILD_COUNT_KEY
+  local _, err = kong_shm:incr(key, 1, 0)
+  if err then
+    log(ERR, "could not record Lua VM allocated memory: ", err)
```

Comment:
It seems to be an incorrect error message.

Reply:
updated